qmcpy/true_measure/matern.py

```diff
@@ -0,0 +1,89 @@
+from .gaussian import Gaussian
+from ..discrete_distribution._discrete_distribution import DiscreteDistribution
+from ._true_measure import TrueMeasure
+from ..discrete_distribution.lattice.lattice import Lattice
+from ..util import DimensionError, ParameterError
+import numpy as np
+from scipy.stats import norm, multivariate_normal
+from sklearn import gaussian_process as gp
```

Comment:
we don't want to introduce a new dependency in sklearn. Try to just implement it yourself and make sure they match